src/deploy/NVA_build/dev.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:8 
```

Comment:
1. What is the use case for this Docker file?
2. Can we add an entry in the Makefile?
3. We can Update the https://github.com/noobaa/noobaa-core/wiki/Building-and-Testing

Reply:
I am ignoring this for now. Will get back to it later.